Gopkg.lock

```diff
@@ -0,0 +1,249 @@
+# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
+
+
+[[projects]]
```

Comment:
Am I the only who thinks toml is ugly?

Reply:
i hate ALL file formats. roughly equally, and for different reasons! 😜